lib/routes/account.js

```diff
@@ -1017,10 +1092,20 @@ module.exports = function (
         }
 
         function createResponse() {
-          return {
-             email: sessionToken.email,
-             verified: sessionToken.emailVerified
-           }
+          return db.sessionTokenWithVerificationStatus(sessionToken.id)
+            .then( function(token) {
+
+              var sessionVerified = !token.tokenVerificationId // tokenVerificationId exists for unverified sessions
```

Comment:
The fact that we need a comment here, is a bit of a red flag warning about future footguns.  Perhaps we could add logic in the SessionToken constructor at [1] that sets an explicit `tokenVerified = true|false` property on the object, rather than having to remember that "tokenVerificationId exists for unverified sessions" in every location that we want to check verification status.

[1] https://github.com/mozilla/fxa-auth-server/blob/master/lib/tokens/session_token.js#L15